generators/server/templates/src/main/java/package/service/_UserService.java

```diff
@@ -11,7 +11,10 @@
 import <%=packageName%>.service.util.RandomUtil;
 import <%=packageName%>.web.rest.vm.ManagedUserVM;
 import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;<% if (databaseType == 'sql' || databaseType == 'mongodb') { %>
+import org.slf4j.LoggerFactory;
+<% if (databaseType == 'sql' || databaseType == 'mongodb') { %>
```

Comment:
You should use <%_ text _%> instead of <% text %>, so you don't add an empty line